public/js/Pull.js

```diff
@@ -24,15 +24,12 @@ _.extend(Pull.prototype, {
          groups.user = null;
 
          signatures.forEach(function(signature) {
-            if (signature.data.active) {
-               groups.current.push(signature);
-               users[signature.data.user.id] = true;
-
-               if (utils.mySig(signature)) {
-                  groups.user = signature;
+            if (!users[signature.data.user.id]) {
+               if (signature.data.active) {
+                  groups.current.push(signature);
+               } else {
+                  groups.old.push(signature);
```

Comment:
Reduce nesting:
```suggestion
               const groups = signature.data.active ? groups.current : groups.old;
               groups.push(signature);
```

FYI: I can not include the last brace in my suggestion so do this by hand please

Reply:
This is fewer lines, but not more readable IMO :shrug:

Reply:
IMO it's more readable. Namely, you don't have to keep the state in your head. Every time I see an if, I mentally note that we are forking the state and need to consider other cases. Between inversion and ternaries, it becomes more clear to me that we are handling + edge case and picking a bucket.

That being said it's up to Andrew here

Reply:
I'm fine with this the way it is.